src/System.Management/tests/System/Management/ManagementClassTests.cs

```diff
@@ -84,5 +83,26 @@ public void EnumerateInstances_For_Win32_LogicalDisk()
                 }
             }
         }
+
+        [ConditionalFact(typeof(WmiTestHelper), nameof(WmiTestHelper.IsElevatedAndNotNanoServer))]
+        public void Create_Delete_Namespace()
+        {
+            using (var rootNamespace = new ManagementClass("root:__namespace"))
+            using (ManagementObject newNamespace = rootNamespace.CreateInstance())
+            {
+                const string NewNameSpace = "CoreFx_Create_Delete_Namespace_Test";
```

Comment:
nit: S in Namespace should not be caps.

if you want you could use sometnhing like `= nameof(Create_Delete_Namespace);`